extensions/amp-loader/0.1/amp-loader.css

```diff
@@ -94,10 +94,6 @@
   margin: 12px;
 }
 
```

Comment:
Does this class need to be removed? Wouldn't this break loaders on the web? CSS comments suggest using `.i-amphtml-new-loader-transparent-on-shim` when transparency is desired. 
https://github.com/ampproject/amphtml/blob/6420be4dfc34244d3e5f3c684971c008a4709cc4/extensions/amp-loader/0.1/amp-loader.css#L109-L116

Reply:
it's actually duplicated earlier in the file, so I think this part of the diff is just a tidiness chance with no functional effect (sorry for mixing it into the PR)

https://github.com/ampproject/amphtml/pull/28896/files#diff-e6ff15832930475ece629f4e3b2214b8L70-L72